File: Corpo.Domain/Models/WodGroupMember.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Corpo.Domain.Models
{
    public class WodGroupMember
    {
        public int Id { get; set; }
        public virtual Exercise Exercise { get; set; }
        public int ExerciseId { get; set; }
        public virtual Modality Modality { get; set; }
        public int ModalityId { get; set; }
        public string Detail { get; set; }
        public string Units { get; set; }
        public string GroupIndex { get; set; }
        public virtual WodMember WodMember { get; set; }
        public int WodMemberId { get; set; }
    }
}

File: Corpo.Web/ClientApp/src/app/domain/wod-group-member.ts
import { Exercise } from "./exercise";
import { Modality } from "./wod/modality";

export class WodGroupMember {
  id?: number;
  exerciseId: number;
  modalityId: number;
  detail: string;
  rounds?: number;
  series?: number;
  time?: number;
  unitType?: string;
  units?: string;
  groupIndex: number;
  intensityType?: string;
  intensityValue?: number;
  staggeredType?: string;
  staggeredValue?: number;
  pauseBetweenRounds?: number;
  pauseBetweenExercises?: number;
  timeWork?: number;
  timeRest?: number;

/*  wodMemberId: number*/
}

export class WodGroupMemberResponse {
  exercise: Exercise;
  modality: Modality;
  rounds?: number;
  series?: number;
  time?: number;
  unitType?: string;
  units?: string;
  groupIndex: number;
  intensityType?: string;
  intensityValue?: number;
  staggeredType?: string;
  staggeredValue?: number;
  pauseBetweenRounds?: number;
  pauseBetweenExercises?: number;
  timeWork?: number;
  timeRest?: number;
}
